lib/src/embedded/compiler/process.test.ts

```diff
@@ -0,0 +1,73 @@
+// Copyright 2020 Google Inc. Use of this source code is governed by an
```

Comment:
Up to you, but I generally find it clearer if tests live in a separate top-level directory.

Reply:
I was unsure of this too. The Google typescript style guide recommends putting tests next to the source. If it starts getting out of hand I'll move them into separate directories.